Resources/Learn Bayes/Description.qml

```diff
@@ -0,0 +1,34 @@
+import QtQuick 		2.12
+import JASP.Module 	1.0
+
+Description
+{
+	name:			"Learn Bayes"
+	title : 		qsTr("Learn Bayes")
+	description:	qsTr("Learning Bayesian Statistics with JASP")
+	icon:			"learn-bayes.svg"
+	version:		"0.14"
+	requiresData:	false
+	author:			"JASP Team"
```

Comment:
Don't you want to be the author?

Reply:
does it add anything than more responsibility? I thought that I'm part of the JASP team as well

Reply:
Nope, just more direct references to your name ;)